index.html

```diff
@@ -132,5 +123,251 @@
+                            <div>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </div>
+                            <div class="year"><p>2015</p></div>
+                        </div>
+                    </div>
+                    <h2 class="project-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
+                    <ul class="tags">
+                        <li class="tags-list"><a class="tags-text" href="">html</a></li>
+                        <li class="tags-list"><a class="tags-text" href="">css</a></li>
+                        <li class="tags-list"><a class="tags-text" href="">javaScript</a></li>
+                    </ul>
+                    <div class="actions">
+                        <button>
+                            <a href="">See&nbspProject</a>
+                        </button>
+                    </div>
+                </div>
+        </div>
+    </main>
+
+    <section class="about-myself">
+        <div class="top">
+            <h1>
+                About <br>
+                Myself
+            </h1>
+            <h2>
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </h2>
+            <div class="social-media">
+                <div class="socials-text">
+                    <div class="supporting-text">
+                        <p>LET’S&nbspCONNECT</p>
+                    </div>
+                </div>
+                
+                <div class="socials-icons-2">
+                    <ul>
+                        <li>
+                            <a href=""><img src="./icons/Github-icon.png" alt=""></a>
+                        </li>
+                        <li>
+                            <a href=""><img src="./icons/Linkedin-icon.png" alt=""></a>
+                        </li>
+                        <li>
+                            <a href=""><img src="./icons/Like-icon.png" alt=""></a>
+                        </li>
+                        <li>
+                            <a href=""><img src="./icons/Twitter-icon.png" alt=""></a>
+                        </li>
+                        <li>
+                            <a href=""><img src="./icons/Mail-icon.png" alt=""></a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <button class="normal-button" href=""><a>Get&nbspmy&nbspresume</a></button>
+        </div>
+
+        <ul class="skill-list">
```

Comment:
- [ Required ] Good job designing this section you did well, however, I noticed some areas that if attended to will make your design even better. Please consider the following observations and adjust.
   
   - There should be a `border-bottom` below the `Frameworks` and `Skills` part 
   - There should be no `margin-left` on the `JavaScript` `CSS` and `HTML5` list
   - The clickable buttons on the list should not have a gray background color

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/221300570-5713be97-d3a6-4599-907f-0d0563785e2e.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/221300252-fff5bbf8-5bd2-4ee4-a8cb-a3ccfea6fe01.png" width="375px" />